samza-api/src/main/java/org/apache/samza/context/SamzaContext.java

```diff
@@ -0,0 +1,21 @@
+package org.apache.samza.context;
+
+/**
+ * Container object for all context provided by the Samza framework to instantiate an application at runtime.
+ */
+public interface SamzaContext {
```

Comment:
How is this different than `Context`?

Reply:
SamzaContext has been removed (see above comment).